src/pricecypher/datasets.py

```diff
@@ -157,6 +158,8 @@ def get_transactions(
         :param datetime end_date_time: When specified, only transactions before this date are considered.
         :param str bc_id: (optional) business cell ID.
             (defaults to 'all')
+        :param list[dict] filter_scopes (optional): When specified, filters for the menitoned scope and scope values are 
```

Comment:
```suggestion
        :param list[dict] filter_scopes: (optional) When specified, filters for the mentioned scope and scope values are
```
The position of the `:` is important here, it needs to come directly after the name of the parameter. Otherwise, IDEs won't properly understand the docstring.